flink-tests/src/test/scala/org/apache/flink/api/scala/migration/StatefulJobWBroadcastStateMigrationITCase.scala

```diff
@@ -75,43 +91,44 @@ object StatefulJobWBroadcastStateMigrationITCase {
       SnapshotSpec.withVersions(
         StateBackendLoader.MEMORY_STATE_BACKEND_NAME,
         SnapshotType.SAVEPOINT_CANONICAL,
-        FlinkVersion.rangeOf(FlinkVersion.v1_8, FlinkVersion.v1_13)))
+        getFlinkVersions.apply(FlinkVersion.v1_8, FlinkVersion.v1_13)
```

Comment:
here as welll: We're filtering in the old code but don't do it in the new code. :thinking: